app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -23,6 +23,10 @@
  * WebViewProvider implementation for creating a Gecko based implementation of IWebView.
  */
 public class WebViewProvider {
+    public static void preload(final Context context) {
+        // Nothing: there's no Gecko preloading (yet?).
```

Comment:
We could startup Gecko here. That's what I wanted to try and for that I needed an early call to the provider too. I might look into this in a new issue. :)